app/models/govuk_publishing_components/audit_applications.rb

```diff
@@ -150,7 +150,7 @@ def find_code_references(file, src, regex)
     end
 
     def clean_file_path(file)
-      file[/(?<=#{Regexp.escape(@path.to_s)}\/)[\/a-zA-Z_-]+.[a-zA-Z.]+/]
+      file[/(?<=#{Regexp.escape(@path.to_s)}\/)[\/a-zA-Z_-]+.[a-zA-Z+.]+/]
```

Comment:
Just double checking that we don't want to escape the `+` as it's a special character?

Reply:
Added, can't hurt.

Reply:
Ah, no, it can hurt. Linter throws up this:

```
Offenses:

app/models/govuk_publishing_components/audit_applications.rb:153:69: C: [Correctable] Style/RedundantRegexpEscape: Redundant escape inside regexp literal
      file[/(?<=#{Regexp.escape(@path.to_s)}\/)[\/a-zA-Z_-]+.[a-zA-Z\+.]+/]
                                                                    ^^

209 files inspected, 1 offense detected, 1 offense autocorrectable
```

There might be a way around this linting issue but I think the original code is fine - the linter seems to think escaping isn't needed, and the code seems to work as intended.

Reply:
Super! Thanks for checking 👍